docs/contributtion/feature-flags.md

```diff
@@ -0,0 +1,67 @@
+# Feature Flags
+
+## Table of Contents
+
+- [Current feature flags](#current-feature-flags)
+- [Turning on feature flags in SASS](#turning-on-feature-flags-in-sass)
+
+The Spirit Design System codebase ships with a number of feature flags. These feature
+flags enable new behavior and styling, allowing you to opt-in to new (and
+sometimes breaking) changes while using the current version. A feature flag may
+be configured in React, SASS, or both.
+
+This section in the Storybook documents each feature flag that is available and
+how to configure it. Folders and stories within this section in Storybook show
```

Comment:
🤔 I wonder if we really want to do another stories

Reply:
I like the way the components are displayed in separate ways. So you can look at them and see what the feature flag is turning on.